### ISSUE TITLE
Using directory in sqlite db_uri failed

### ISSUE THREAD

**Issue by jrmi** [tortoise/tortoise-orm#60]
If i try to use a db_uri with subdirectory like "sqlite://data/db.sqlite" I get the following error:

```python
Traceback (most recent call last):
  File "buggy.py", line 23, in <module>
    loop.run_until_complete(init())
  File "/usr/lib/python3.6/asyncio/base_events.py", line 468, in run_until_complete
    return future.result()
  File "buggy.py", line 16, in init
    modules={'models': ['buggy']}
  File ".venv/lib/python3.6/site-packages/tortoise/__init__.py", line 301, in init
    await cls._init_connections(connections_config, _create_db)
  File ".venv/lib/python3.6/site-packages/tortoise/__init__.py", line 166, in _init_connections
    await connection.create_connection()
  File ".venv/lib/python3.6/site-packages/tortoise/backends/sqlite/client.py", line 45, in create_connection
    await self._connection._connect()
  File ".venv/lib/python3.6/site-packages/aiosqlite/core.py", line 190, in _connect
    self._connection = await self._execute(self._connector)
  File ".venv/lib/python3.6/site-packages/aiosqlite/core.py", line 183, in _execute
    raise result
  File ".venv/lib/python3.6/site-packages/aiosqlite/core.py", line 155, in run
    result = fn()
  File ".venv/lib/python3.6/site-packages/aiosqlite/core.py", line 304, in connector
    return sqlite3.connect(str(database), **kwargs)
sqlite3.OperationalError: unable to open database file

```
I use tortoise-orm version 0.10.9.

You can reproduce by copy pasting the following code in a `buggy.py` file:

```python
import asyncio
from tortoise import Tortoise
from tortoise.models import Model
from tortoise import fields

class Tournament(Model):
    id = fields.IntField(pk=True)
    name = fields.TextField()

    def __str__(self):
        return self.name

async def init():
    await Tortoise.init(
            db_url='sqlite://data/data_test.sqlite',
        modules={'models': ['buggy']}
    )
    # Generate the schema
    await Tortoise.generate_schemas()

loop = asyncio.get_event_loop()

loop.run_until_complete(init())
```
Subdirectory should be supported in Tortoise-orm as `aiosqlite` supports them.

As workaround I use the config object (which seems to be yet undocumented).

```python
class Tournament(Model):
    id = fields.IntField(pk=True)
    name = fields.TextField()

    def __str__(self):
        return self.name

config = {
    'connections': {
        'default': {
            'engine': 'tortoise.backends.sqlite',
            'credentials':{
                'file_path': 'data/test.sqlite'
            }
        },
    },
    'apps': {
        'models': {
            'models': ['buggy'],
            'default_connection': 'default',
        }
    }
}

async def init():
    await Tortoise.init(
        config=config
    )
    # Generate the schema
    await Tortoise.generate_schemas()

loop = asyncio.get_event_loop()
loop.run_until_complete(init())
```

**Comment by grigi** [comment]
... I will need to document the whole config object ...
And those should be identical...

**Comment by grigi** [comment]
Thanks for reporting.
Replicated issue and fixed. I bundled it as part of #59 which I think is likely to be the next release.

**Comment by grigi** [comment]
Ok, I checked, we do have documentation for the config object:
https://tortoise-orm.readthedocs.io/en/latest/setup.html#tortoise.Tortoise.init

Just for interest sake, how did you look for it?
How can we structure the documentation to make finding something like this easier?

**Comment by jrmi** [comment]
Ok great for #59 !

I looked this page https://tortoise-orm.readthedocs.io/en/latest/databases.html and i miss the setup part but I looked the sources and found the doc you've just pointed me. Thanks.

May be merging the chapter "setup" and "database" ? Or making a cross reference ?

**Comment by jrmi** [comment]
#59 actually fix this bug ! Thanks.

**Comment by grigi** [comment]
Will be fixed for v0.10.10
